### PR TITLE
[ELI5] Tweak wording on the signup screen

### PR DESCRIPTION
**Pull request by gaearon**
People get confused here unsure what "handle" is. "Input your handle" is also confusing.

The first commit replaces the wordings.

Screenshot 2024-12-17 at 00 02 59

This matches our usage of "username" on the login screen.

The second commit is a bit unusual — I'm updating all translations as if this was the wording all along. That's to avoid regressions to translations just before the release. It's fine to update them later (if needed per language) since conceptually the phrasing is the same and the meaning has not changed.

It's a little inconsistent with what we have in Settings. Maybe in the future we'll change it there too. But for now let's make sure people are able to create an account.

## Test Plan

Verify above for English.

Then `yarn intl:build` and verify translations have not regressed and show old strings:

Screenshot 2024-12-17 at 00 01 50

### CODE REVIEW

**Comment by gaearon** [comment]
got verbal ok from @pfrazee

**Review by mozzius — COMMENTED** [review]
code lgtm for what it's worth
